File: Exam2/Ruby/shape.rb
# Base class

class Shape

	def initialize (aColor, aBorder) 
		@color = aColor
		@border = aBorder
	end

	def draw
		puts "No information to draw myself."
	end

	def getColor
		@color
	end

	def getBorder
		@border
	end


end

File: Exam2/C++/shape.cpp
#include "shape.h"
#include <iostream>
#include <string>
using namespace std;

Shape::Shape(string aColor, float aBorder)
{
		color = aColor;
		border = aBorder;
}

string Shape::getColor()
{
	return color;
}

float Shape::getBorder()
{
	return border;
}

void Shape::draw()
{
	cout << "No information to draw myself." << endl;
}
